builtin/providers/scaleway/resource_scaleway_server.go

```diff
@@ -90,7 +98,20 @@ func resourceScalewayServerCreate(d *schema.ResourceData, m interface{}) error {
 	server.CommercialType = d.Get("type").(string)
 
 	if bootscript, ok := d.GetOk("bootscript"); ok {
-		server.Bootscript = String(bootscript.(string))
+		bootscript_id := bootscript.(string)
+
+		bootscripts, err := scaleway.GetBootscripts()
```

Comment:
terraform 0.7.7 added a data source so we don't have to overload the meaning here anymore. please revert this chunk